### PR TITLE
Add aero-sample-apps package

### PR DESCRIPTION
**Pull request by avinash-palleti**
Adding aero-sample-apps package which depends on ros-sample-apps

### CODE REVIEW

File: aero-sample-apps/aero-sample-apps [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Section: misc
+Priority: important
```
Comment (anitha-suresh-zz) [inline aero-sample-apps/aero-sample-apps]: I think priority can be 'optional'

File: aero-sample-apps/aero-sample-apps [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Homepage: https://github.com/intel-aero/sample-apps
+Standards-Version: 3.9.2
+
+Package: aero-sample-apps
+Version: 1.0.1
+Maintainer: Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>
+Depends: ros-kinetic-aero-flight-modes, ros-kinetic-aero-offboard-ctrl, ros-kinetic-aero-telemetry-simple, ros-kinetic-aero-fly-mission, ros-kinetic-aero-takeoff-land
+Description: Collection of Sample applications to run on Aero.
```
Comment (anitha-suresh-zz) [inline aero-sample-apps/aero-sample-apps]: "Sample applications to run on Aero" -> Sample applications can be run on Aero

File: aero-sample-apps/aero-sample-apps [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Section: misc
```
Comment (anitha-suresh) [inline aero-sample-apps/aero-sample-apps]: This indicates that Debian package is stored at the 'misc' folder in the site where we are hosting the package. We need to re-check on this.

File: aero-sample-apps/aero-sample-apps [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+Priority: optional
+Homepage: https://github.com/intel-aero/sample-apps
+Standards-Version: 3.9.2
+
+Package: aero-sample-apps
+Version: 1.0.1
+Maintainer: Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>
+Depends: ros-kinetic-aero-flight-modes, ros-kinetic-aero-offboard-ctrl, ros-kinetic-aero-telemetry-simple, ros-kinetic-aero-fly-mission, ros-kinetic-aero-takeoff-land
+Description: Collection of Sample applications can be run on Aero.
```
Comment (anitha-suresh-zz) [inline aero-sample-apps/aero-sample-apps]: Collection of Sample applications *that* can be run on Aero.

**Comment by anselmolsm** [comment]
IMO this meta package should be added with / after the packages it installs

**Review by anselmolsm — CHANGES_REQUESTED** [review]
Add a note in the commit indicating these packages take care of their own packaging (i.e. they deal with the debian dir magic)

**Comment by avinash-palleti** [comment]
@anselmolsm Done.

**Review by anitha-suresh-zz — COMMENTED** [review]
Spell check : avialble

**Pull request merged**